asynction/playground_docs.py

```diff
@@ -29,8 +29,10 @@ def make_raw_spec_view(spec: AsyncApiSpec) -> View:
     return lambda: jsonify(spec.to_dict())
 
 
-def make_docs_blueprint(spec: AsyncApiSpec, url_prefix: Path) -> Blueprint:
-    bp = Blueprint("asynction_docs", __name__, url_prefix=str(url_prefix))
+def make_docs_blueprint(
```

Comment:
@dedoussis I'm not sure what to do about the name here. I am passing in the `spec.info.title` but that only works if your title only contains the correct characters. I think for the multi API it may be a situation where the use of a new Jinja template that can handle multiple specs would be better?